src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1222,8 +1241,17 @@ void Visitor::visit(Parser::PrologueContext* ctx) {
 }
 
 // ____________________________________________________________________________________
-void Visitor::visit(const Parser::BaseDeclContext* ctx) {
-  reportNotSupported(ctx, "BASE declarations are");
+bool Visitor::isAbsoluteIri(std::string_view string) {
+  return ctre::starts_with<"<[A-Za-z]*[A-Za-z0-9+-.]:">(string);
```

Comment:
As stated above, chec, for all the quirks already dealt with in the `TurtleParser` and use as much common code as possible.